### ISSUE TITLE
User can play tracks from available_markets that do not include his ?

### ISSUE THREAD

**Issue by hycday** [spotify/web-api#437]
Issue found on March, 3rd 2017.

#### Endpoint(s):
* `GET /v1/search`

#### Scope(s):
* None (application is not using authentication)
* We test to play a track from a Spotify Premium FR account (web and Desktop)

#### Steps to reproduce:
1. do a search with `query`=`Taha`, `type`=`track`, `market`=`JP`, `offset`=`0`, `limit`=`1` (?query=Taha&type=track&market=JP&offset=0&limit=1)
2. continue until you find a track with "available_markets" not containing FR
2.a or consider the track with `id` `3Fu2ad8Uu7IKexgwLspW22` for example

#### Expected behaviour:
The track with `id` `3Fu2ad8Uu7IKexgwLspW22` only has the following available_markets:
```
                            [available_markets] => Array
                                (
                                    [0] => HK
                                    [1] => ID
                                    [2] => JP
                                    [3] => MY
                                    [4] => PH
                                    [5] => SG
                                    [6] => TW
                                )
```

The user from FR tries to play the track from https://open.spotify.com/track/3Fu2ad8Uu7IKexgwLspW22 or from spotify:track:3Fu2ad8Uu7IKexgwLspW22
Excpected behavior is unknown, would/should the track not play ? or fail to an alternative track ? or display a message that the track is not available in the market of the user ? 

#### Actual behaviour:
It actually plays the tracks in Web and Desktop (does the fact that the account is Premium change anything?)

To sum up, the question is : 
if a search is done with a market parameter equal to an ISO 3166-1 alpha-2 country code (NOT a from_token), would the result(s) be playable anywhere ?
if a search is done WITHOUT a market parameter, would the result(s) be tracks always playable anywhere ? 
I want to find tracks playable everywhere (or maybe have a fallback).

PS : when I say everywhere/anywhere, I mean in all launched markets

**Comment by hughrawlinson** [comment]
Hi - this track is playable in `FR` because of (Track Relinking](https://developer.spotify.com/web-api/track-relinking-guide/). In some cases, it may not be possible to find a fallback track for a specific market (i.e. there are some tracks that are only playable in certain territories, that do not have linked tracks for other territories). Hope that clears things up!
